web-nfc/nfc_recordType_opaque-manual.https.html

```diff
@@ -21,13 +21,13 @@
 
 setup({ explicit_timeout: true });
 
-let desc = "Test that nfc.push and nfc.watch succeed when recordType is set to 'opaque'."
-let watchOptions = {
+const desc = "Test that write and read data succeed when recordType is set to 'opaque'."
+const watchOptions = {
```

Comment:
Nit: s/watchOptions/readOptions
Please update the rest with same name. :)

Reply:
Updated